app/_includes/gateway-support.html

```diff
@@ -0,0 +1,40 @@
+
+Kong Gateway {{ include.version }} supports the following deployment targets:
+
+<table style="display: table;" width="100%">
+  <thead>
+    <tr>
+      <th>OS</th>
+      <th>Version</th>
+      <th>Artifacts</th>
+      <th>ARM support</th>
+      <th>FIPS compliant</th>
+      <th>EOL</th>
```

Comment:
Here we use `EOL`, but elsewhere we use "End of Full Support" and "End of Sunset Support". Consistency and definitions would help the reader here.